project/templates/configs/springboard.ini

```diff
@@ -0,0 +1,97 @@
+[app:springboardfrontend]
+use = egg:springboard_{{ app_type }}
+unicore.content_repos = unicore_frontend_{{app_type}}_{{country}}
+
+pyramid.default_locale_name = {{default_language}}
+
+featured_languages =
+    {{default_language}}
```

Comment:
@Rizziepit I was meant to remove this. We currently don't specify featured languages anymore in Mission Control so this should be blank. The fallback should be `current language + 2 other`